x-pack/plugins/security/server/lib/authentication/providers/saml.ts

```diff
@@ -533,4 +604,12 @@ export class SAMLAuthenticationProvider {
       `${this.options.basePath}/api/security/v1/saml`
     );
   }
+
+  /**
+   * Logs message with `debug` level and saml/security related tags.
+   * @param message Message to log.
+   */
+  private debug(message: string) {
+    this.options.log(['debug', 'security', 'saml'], message);
+  }
 }
```

Comment:
__note:__ if we need to add anything else to saml we may want to create a dedicated folder for it and split this file..... It's getting too big.